tests/test_alert.py

```diff
@@ -57,6 +58,76 @@
     "template": "custom",
 }
 
+receiver_config_bearer_env = {
```

Comment:
Most of the not tested lines are part of the code that should not be reached since the validation is performed with the schema validation:
- 66-67
- 115-116
- 151
- 158
- 205

This is in case the file reading fails for unknown reason (FileNotFoundError tested separately).
The approach mirrors the code already used to load a file in lines 277-288. 
- 177-178

These are lines not concerning this pull request:
- 241-243
- 285-286